src/views/Home.vue

```diff
@@ -0,0 +1,107 @@
+<template>
+<div id="app">
+
+<div class="body">
+  <div class="step-box">
+    <span>STEP1</span>
+    <h4>お客様情報を入力してください</h4>
+  </div>
+  <div class="question-box">
+    <p>-性別-</p>
+    <div class="boxes">
+      <label><input type="radio" v-model="radio" @click="genderisaMan"  value="yes" >男性</label>
+      <label><input type="radio" v-model="radio" @click="genderisaWoman" value="no" >女性</label>
+    </div>
+  </div>
+  <br>
+  <div class="question-box">
+    <p>-生年月日-</p>
+    <div class="boxes">
+    <select id="year" v-model="year">
+    <option v-for="number in years" :key="number.years" :value="number.year">{{ number.label }}</option>
+    </select>
+    <label>年</label>
+    <select v-model="month" @change="get_days">
+    <option v-for="n in 12" :value="n" :key="n">{{ n }}</option>
+    </select>月
+    <select name='day' v-model="day">
+    <option v-for="n in 31" :value="n" :key="n">{{ n }}</option>
+    </select>日
+    </div>
+  </div>             
+</div>
+
+<router-link to="/about"><button class="button is-primary" id="next-btn">次へ進む</button></router-link>
+
+</div>
+
+</template>
+
+<script>
+import {  mapGetters,  } from 'vuex'
+export default {
+  data() {
+    return {
+    
+      
+      date: 1,
+      radio:'',
+      radio2:'',
+      radio3:'',
+      years: []
+    };
+  },
+  mounted() {
+    this.years = this.genereate();
+  },
+  
+  methods: {
+    genereate() {
+      const years = [];
+      for (let y=2020; y>1920; y--) {
```

Comment:
以下のようにスペースを空けるのが一般的です。
他の箇所ではできているところもあるので、統一しましょう。
```suggestion
      for (let y = 2020; y > 1920; y--) {
```